package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "balancer-core",
-  "version": "0.0.7",
+  "name": "balancer-cow-amm",
+  "version": "1.0.0",
   "private": true,
-  "description": "Balancer Core Contracts and ABI",
-  "homepage": "https://github.com/balancer-labs/balancer-core#readme",
+  "description": "Balancer CoW AMM",
+  "homepage": "https://github.com/balancer-labs/cow-amm#readme",
```

Comment:
this 404s, is this currently private and will be made public? will balancer create it later?